packages/grafana-ui/src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,38 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+import { useTheme2 } from '../../themes';
+
+interface Props {
+  direction?: 'vertical' | 'horizontal';
+}
+
+export const Divider = ({ direction = 'horizontal' }: Props) => {
+  const theme = useTheme2();
+  const styles = getStyles(theme);
+
+  if (direction === 'vertical') {
+    return <div className={styles.verticalDivider}></div>;
+  } else {
+    return <hr className={styles.horizontalDivider} />;
+  }
+};
+
+Divider.displayName = 'Divider';
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    horizontalDivider: css`
+      border-bottom: 1px solid ${theme.colors.border.weak};
```

Comment:
The `<hr/>` element has a `border-top` already set, which you need to override. Otherwise the vertical and horizontal end up being different colors 
![image](https://github.com/grafana/grafana/assets/1438972/0415d71b-2921-43f2-9ad2-e1b9e0c22246)

Reply:
Thank you, great advice, I made the necessary changes now.